java-squid/src/main/java/org/sonar/java/ast/parser/TreeFactory.java

```diff
@@ -1120,7 +1120,10 @@ public InstanceOfTreeImpl newInstanceofExpression(InternalSyntaxToken instanceof
     return new InstanceOfTreeImpl(instanceofToken, type);
   }
 
-  public VariableTreeImpl receiverParameterId(InternalSyntaxToken thisToken) {
+  public VariableTreeImpl receiverParameterId(Optional<List<Tuple<InternalSyntaxToken, InternalSyntaxToken>>> optional, InternalSyntaxToken thisToken) {
+    if(optional.isPresent()) {
+      // FIXME qualified id of outer class for receiver type.
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1134)